media/com_redshop/css/redshop.admin.min.css

```diff
@@ -18318,7 +18318,9 @@ i.tools {
 
 #adminForm .filterTool .filterItem {
     display: inline-block;
-    margin: 10px 4px 4px;
```

Comment:
@hoangvan149dhv This is not MIN CSS file. Please make sure this file is minified before using.